### ISSUE TITLE
Feature request: Draftail Strikethrough

### ISSUE THREAD

**Issue by keeganland** [ubyssey/ubyssey-dispatch-theme#1002]
Our editors would like to be able to do strikethroughs in the Rich Text editors in Wagtail.

Wagtail implements its Rich Text editor using Draftail. Plugins etc. can be added to Draftail, which is itself based on Draft.JS. The following documentation exists for writing or installing extensions to Draftail:

https://docs.wagtail.org/en/stable/extending/extending_draftail.html

Mention of enabling Draftail strikethrough on this page:

https://www.draftail.org/docs/formatting-options

It seems it should not be difficult to flip a few switches to get it enabled, we just need to read this documentation to see explicitly which ones

**Comment by brittkhat** [comment]
move to a later date and lower priority

**Comment by FireBoyAJ24** [comment]
Can I take up this issue?

**Comment by FireBoyAJ24** [comment]
I have made a new branch called the "strikethrough-rich-feature". I only added a wagtail_hooks.py file with my current rich text bar:
![image](https://user-images.githubusercontent.com/63443017/204121920-83ff1a12-fba9-4c33-bd4a-932010d19912.png)

I am struggling to move the strikethrough button to the left side of the rich text bar. An idea is to completely remove the rich text bar and then append each button in a specific order.
